### ISSUE TITLE
Add support for Svelte language

### ISSUE THREAD

**Issue by benaltair** [thomaswilburn/Caret#756]
One of the fastest growing compilers - [Svelte](https://svelte.dev/) ([repo](https://github.com/sveltejs/svelte)) - uses `.svelte` files and it would be great to support them. The basic syntax isn't much different than vanilla `html` and `js`.  

### Description

At the very least, it would be great to have syntax highlighting. A much great step would be to have native language tooling added using [Svelte language tools](https://github.com/sveltejs/language-tools).

### System details

Chrome OS

**Comment by thomaswilburn** [comment]
Do you know if Svelte support has been added to Ace? We rely on it for syntax highlighting.

**Comment by benaltair** [comment]
@thomaswilburn I don't think so. I've submitted an issue there now (https://github.com/ajaxorg/ace/issues/4491)
